android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -43,7 +45,7 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
      * modules in case a full-custom implementation is needed. A default implementation is provided
      * out-of-the-box.
      */
-    var pictureInPictureHandler = RNPictureInPictureHandler(context)
+    private var pictureInPictureHandler = RNPictureInPictureHandler(context)
```

Comment:
Do you know why we have one instance of pictureInPictureHandler on the `RNPlayerViewManager` and one on the `RNPlayerView`? This feels like an oversight to me and i think this one should be removed probably.

Reply:
Good catch!
I removed the stored instance in `RNPlayerViewManager` and reusing the existing or creating a new one in 7872455